### ISSUE TITLE
Feature Request: Save Password Site Filter

### ISSUE THREAD

**Issue by SniperFodder** [bitwarden/browser#275]
I work with a lot of IP addresses where I don't want bit warden to prompt me to save the password. It would be nice if I could add a custom filter for the prompt on which web addresses Bit Warden will ignore for password saving.

**Comment by kspearrin** [comment]
We already provide a "Never" option on the prompt. Is this not enough?

**Comment by SniperFodder** [comment]
Yes, but I have to do it for each new IP. If I only worked with 5 IP's, sure, But I'm working with close to 1000 different IPs.
